SparkPost/support-docs
# WE-164 style side navigation menu

## What Changed
- Adds styling and expanding functionality to the side navigation

## How to verify
- Run locally or visit the deploy link to verify the styling and functionality are correct
